scripts/services/storage

```diff
@@ -19,7 +19,7 @@ else
 		exit 1
 	else
 		if [ -x ${FOGLAMP_ROOT}/services/foglamp.services.storage ]; then
-			export LD_LIBRARY_PATH=${FOGLAMP_ROOT}:/usr/local/foglamp/lib
+			export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${FOGLAMP_ROOT}:/usr/local/foglamp/lib
```

Comment:
Should it be ${FOGLAMP_ROOT}/lib?

Reply:
I have added the LD_LIBRARY_PATH and I haven't change what was already there, 
the correct setup is made by the foglamp script, sample  line 104 from foglamp script :  

libdir=${FOGLAMP_ROOT}/cmake_build/C/lib; [ -d ${libdir} ] && LD_LIBRARY_PATH=$(echo $LD_LIBRARY_PATH | sed "s|${libdir}||g") && export LD_LIBRARY_PATH=${libdir}:${LD_LIBRARY_PATH} && libPathSet=1

Reply:
Yes, I understand that existing value is wrong and is unrelated to your change, but since you are changing around this area, please go ahead and make this change also.